GUI/builder.py

```diff
@@ -158,6 +160,6 @@ def builder(folder, NGRAMS, GAMMA, BATCH_SIZE, LEARNING_RATE, EMBED_DIM, EPOCHS,
     Statistics.test_cat_val = test_comp
 
     # Saves the model
-    torch.save(model, root + "model")
-    torch.save(train_dataset.get_vocab(), root+"vocab")
-    return Statistics
\ No newline at end of file
+    torch.save(model, "model")
```

Comment:
Are we confident that the full path won't be needed here? Anyway way for the "model" variable to be corrupted/changed in a way that's not intended?

Reply:
The path is correct since the current directory has been changed to the specified model using os.chdir() in the select folder function of methods.py. I made sure to test the current directory throughout various points using print. As of right now, the model variable shouldn't be corrupted since it's in the models directory

Reply:
Great!